pages/linux/phpdismod.md

```diff
@@ -0,0 +1,11 @@
+# phpdismod
+
+> Disable PHP extensions on Debian-based OSes.
+
```

Comment:
Is there anywhere we could link to in the more information link?

Reply:
I could not find any official page (it does not even have a man page) :slightly_frowning_face:

Reply:
No problem :-)